typescript/sdk/src/providers/providerBuilders.ts

```diff
@@ -6,11 +6,13 @@ import { RpcProvider as StarknetRpcProvider } from 'starknet';
 import { createPublicClient, http } from 'viem';
 import { Provider as ZKProvider } from 'zksync-ethers';
 
+import { HyperlaneModuleClient } from '@hyperlane-xyz/cosmos-sdk';
```

Comment:
Does this have a large impact on bundle size? Does it bring any issues with nodejs dependencies not available on the browser?

Reply:
So it mainly consists of the `@cosmjs/stargate` package where you can find the bundle size here: https://bundlephobia.com/package/@cosmjs/stargate@0.33.1

It is required since it is the only one with which we can support cosmos from TypeScript. It does not have any nodejs dep issues on the browser since everything is imported and ready to use in our cosmos-sdk package